test/ui-test/testSuites/suite_wallet/tst_wallet/test.feature

```diff
@@ -2,56 +2,65 @@ Feature: Status Desktop Wallet
-        And the user has a positive balance of ETH
-        And the user has a positive balance of STT
-        # And the collectibles are listed for the one
-        And the transactions are listed for the added account
-
-
-	Scenario: User can manage a list of accounts
-        When the user adds watch only account with AccountWatch and 0x8397bc3c5a60a1883174f722403d63a8833312b7
-        Then the new account AccountWatch is added
-        When the user imports a private key with AccountPrivate and TesTEr16843/!@00 and 8da4ef21b864d2cc526dbdb2a120bd2874c36c9d0a1fb7f8c63d7f7a8b41de8f
-        Then the new account AccountPrivate is added
-        When the user imports a seed phrase with AccountSeed and TesTEr16843/!@00 and pelican chief sudden oval media rare swamp elephant lawsuit wheat knife initial
-        Then the new account AccountSeed is added
-        When the user generates a new account with AccountGenerated and TesTEr16843/!@00
-        Then the new account AccountGenerated is added
-        When the user deletes the account AccountGenerated
-        Then the account AccountGenerated is not in the list of accounts
-        When the user opens app settings screen
+        When the user imports a seed phrase with "one" and "TesTEr16843/!@00" and "pelican chief sudden oval media rare swamp elephant lawsuit wheat knife initial"
+        Then the new account "one" is added
+        And the user has a positive balance of "ETH"
+        And the user has a positive balance of "STT"
+        # And the collectibles are listed for the on        And the transactions are listed for the added account
+
+	Scenario: The user can manage a watch only account
+        When the user adds watch only account with "AccountWatch" and "0x8397bc3c5a60a1883174f722403d63a8833312b7"
+        Then the new account "AccountWatch" is added
+
+    Scenario: The user imports a private key
+        When the user imports a private key with "AccountPrivate" and "TesTEr16843/!@00" and "8da4ef21b864d2cc526dbdb2a120bd2874c36c9d0a1fb7f8c63d7f7a8b41de8f"
+        Then the new account "AccountPrivate" is added
+
+	Scenario: The user generates a new account from wallet
+        When the user generates a new account with "AccountGenerated" and "TesTEr16843/!@00"
+        Then the new account "AccountGenerated" is added
+
+        When the user deletes the account "AccountGenerated"
+        Then the account "AccountGenerated" is not in the list of accounts
+
+    Scenario: The user edits the default account
+        Given the user opens app settings screen
         And the user opens the wallet settings
-        And the user selects the default account
-        And the user edits default account to Default name and #FFCA0F color
-        Then the new account with name DefaultStatus account and color #FFCA0F is updated
+        When the user selects the default account
+        And the user edits default account to "Default" name and "#FFCA0F" color
```

Comment:
This may need clarified given the expected result is given as "DefaultStatus account" but the edit is only "Default".